docs/interfaces/CC++.rst

```diff
@@ -42,6 +46,10 @@ Problem data can be updated without executing the setup again using the followin
 
 .. doxygenfunction:: osqp_update_lin_cost
 
+.. doxygenfunction:: osqp_update_lower_bound
+
+.. doxygenfunction:: osqp_update_upper_bound
+
```

Comment:
Similar comment as above. I think we can just keep `osqp_update_bounds` and set the corresponding arguments to `NULL` if we only want to update one bound.

Reply:
Done. Not changing this in master just yet.